perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -1879,6 +1879,9 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 
 	/*#
 	 * Get page of members from the given vo, with the given attributes.
+	 * Query parameter specifies offset, page size, sorting order, sorting column, statuses in vo and string to search
+	 * members by (by default it searches in names, emails, logins of member or other attributes based on perun
+	 * configuration), last two parameters are optional and by default it finds all members.
```

Comment:
`@param query query` won't generate anything meaningful in the API docs. We need to specify usage of `MembersPageQuery` and you need to add support for its definition in docs generating script in `perun-utils`.

Reply:
Ok, I've edited parameters and added `MembersPageQuery` and `Paginated<RichMember>` to `perun-utils`.